.ipynb_checkpoints/Deque.py

```diff
@@ -0,0 +1,80 @@
+import sys
+
+
+class Deque(object):
+    def __init__(self):
+        self.Size = 100
+        self.array = [0] * 100
+        self.Front = 0
+        self.end = 0
+        self.actualSize = 0
+
+    def push_front(self, n):
+        self.Front -= 1
+        self.array[self.Front] = n
```

Comment:
if self.Front == 0, you will get an exception here

Reply:
Clear